src/locals/pt-br.json

```
@@ -0,0 +1,141 @@
+{
+  "onboarding_screen": {
+    "auto_thought_ex": "Me esqueci da festa do Jorge! Ele deve me odiar.",
+    "challenge_ex": "O Jorge sabe que não consigo ir a todas as festas. Eu não o odiaria caso se esquecesse da minha. Tenho que me considerar tão bem quanto considero os outros.",
+    "alt_thought_ex": "É verdade que me esqueci da festa do Jorge mas posso pedir desculpa e ele compreenderá.",
+    "cog_distortion": { "label": "Leitura da mente", "slug": "leitura-mente" },
+    "intro": {
+      "line1": "Bem vindo ao",
+      "emphasis1": "quirk",
+      "line2": ", uma",
+      "emphasis2": "app",
+      "line3": " de terapia cognitivo comportamental (TCC)."
```

What if we Title Case "Terapia Cognitivo Comportamental"?
```suggestion
      "line3": " de Terapia Cognitivo Comportamental (TCC)."
```